packages/contracts/test/test-utils/psp/wrappers.ts

```diff
@@ -54,10 +57,12 @@ export async function applyInstallation(
     )
   );
 
-  const event = await findEvent<InstallationAppliedEvent>(
-    tx,
-    'InstallationApplied'
-  );
+  const eventName = 'InstallationApplied';
+  const event = await findEvent<InstallationAppliedEvent>(tx, eventName);
+  if (!event) {
+    throw new Error(`Failed to get ${eventName} event`);
+  }
+
```

Comment:
a new function called `findEventOrFail()` would make sense so you don't have to repeat the same pattern multiple times

Reply:
This is what I did already inside. I am wondering how I ended up doing this.

Reply:
Removed redundant catches in 1b575a15e5fa219fcb9a8fb8bcb0ee9fc7e93b07